pages/careers/_open-positions.html

```diff
@@ -1,8 +1,28 @@
 <div class="row row-page-header">
   <div class="col-xs-12">
     <h2>Open Positions</h2>
-    <p>We are always ready to add the next Grunt, but as a 100% bootstrapped company, we only hire when our revenue supports it, so timing matters. If you're inspired by what you've seen and believe in a little serendipity, we would love to hear from you. Here are the position we're looking for:</p>
+    <p>We are always ready to add the next Grunt, but as a 100% bootstrapped company, we only hire when our revenue supports it, so timing matters. If you're inspired by what you've seen and believe in a little serendipity, we would love to hear from you. Here are the position we're currently looking for:</p>
     <h3>Senior Full-Stack Engineer</h3>
+    <h4>What You'll Do</h4>
+    <ul>
+      <li><strong>Work on just about every aspect of the Gruntwork stack.</strong> Write Go-based CLI tools, work on Serverless apps in Node, write Terraform modules, write Python scripts and Bash scripts, do frontend development, learn the newest tech from AWS, invent new ways of managing infrastructure. We're looking for true full-stack engineers who can go deep in any area. We're less concerned with prior experience than we are with demonstrated ability to go deep when necessary.</li>
+    </ul>
+    <h4>Your Ideal Background</h4>
+    <ul>
+      <li>Demonstrated experience building good software.</li>
+      <li>Bonus points for visible open source work.</li>
+      <li>Bonus points for any technical presentations you've done in the past.</li>
+      <li>Bonus points for a blog.</li>
+      <li>Bonus points for something in your profile that makes us smile.</li>
+      <li>To be clear, you don't have to be good at everything. Our most important criteria are:
+        <ul>
+          <li>Strong cultural fit</li>
+          <li>You communicate well.</li>
+          <li>You have a broad and strong technical foundation.</li>
+          <li>You have a passion for writing software and a track record that shows it.</li>
```

Comment:
Again, what do you see as the benefit of writing this? Who is going to read this and say "oh dear, I don't communicate well, so I probably shouldn't apply."

Reply:
I've seen great candidates disqualify themselves before so I wanted to make it apparent that we're not looking for perfection, just some foundational traits. That being said, to your point, "good communication" is definitely a job posting platitude. 

After thinking it over, I decided to remove these. Check out the latest updates in #37 and let me know what you think. I'll push the changes live but leave the PR open.